sdk/communication/tests.yml

```diff
@@ -5,4 +5,9 @@ extends:
   parameters:
     ServiceDirectory: communication
     EnvVars:
-      AZURE_COMMUNICATION_TEST_MODE: Record
+      AZURE_SUBSCRIPTION_ID: $(acs-subscription-id)
+      COMMUNICATION_LIVETEST_CONNECTION_STRING: $(communication-livetest-connection-string)
+      PHONE_NUMBER: $(communication-livetest-phone-number)
```

Comment:
nit: can we make sure we keep the env variables consistent with others. e.g. put AZURE in front of them